homework.js

```diff
@@ -0,0 +1,161 @@
+//1-----------------
+let sum = [1, 2, 3, 4, 5].reduce(add,0); 
```

Comment:
Improper form, the function should be called summation and takes one argument that is a number